terraform-organize-configuration-workshop/index.json

```diff
@@ -0,0 +1,58 @@
+{
+  "title": "Organize Terraform Configuration",
+  "description":
+    "Learn to organize Terraform configuration.",
+  "time": "30 minutes",
+  "icon": "fa-terraform",
+  "details": {
+    "steps": [
+      {
+        "title": "Create Terraform Configuration",
+        "text": "step1.md"
+      },
+      {
+        "title": "Separate Configuration Files",
+        "text": "step2.md"
+      },
+      {
+        "title": "Separate Configuration Directories",
+        "text": "step3.md"
+      },
+      {
+        "title": "Separate State With Workspaces",
+        "text": "step4.md"
+      },
+      {
+        "title": "Refactor With Modules",
+        "text": "step5.md"
+      }
+    ],
+    "intro": {
+      "text": "intro.md",
+      "code": "courseBase.sh",
+      "credits": "https://learn.hashicorp.com/terraform"
+    },
+    "finish": {
+      "text": "finish.md",
+      "credits": "https://learn.hashicorp.com/terraform"
+    },
+    "assets": {
```

Comment:
The assets feature is flaky. I would clone a git repo instead.

Reply:
+1, will do after beta teach